ddtrace/profiling/profiler.py

```diff
@@ -32,6 +32,13 @@ def _build_default_exporters():
     return exporters
 
 
+def _get_service_name():
+    for service_name_var in ("DD_SERVICE", "DD_SERVICE_NAME", "DATADOG_SERVICE_NAME"):
+        service_name = os.environ.get(service_name_var)
+        if service_name is not None:
+            return service_name
+
```

Comment:
Are we okay with relying on the implicit `return None` here for the service name if one is not found? It was like this formerly so just a non-blocking thought.

Reply:
Yes, that's handled in the http profiler.